src/main.rs

```diff
@@ -204,4 +212,7 @@ fn main() {
       error!("{}", err);
       std::process::exit(1);
     });
+
+  // Start the Tokio event loop
+  d.rt.run().expect("err");
```

Comment:
Ok for now, I think, fingers crossed. 
But we'll have to contain this soon or we might as well rename to tokio,js already.